chat-widget/samples/javascript-sample/getUnreadMessageCount.js

```diff
@@ -10,4 +10,14 @@ export const getUnreadMessageCount = async () => {
             window.document.title = title;
         }
     });
+};
+
+export const registerVisibilityListener = async () => {
```

Comment:
Isn't this listener already there in the script layer?

Reply:
This will be helpful in testing the changes on chat-widget codebase without going to LCW V2 (LiveChatWidget codebase) .